Project.toml

```diff
@@ -4,10 +4,12 @@ authors = ["Beacon Biosignals, Inc."]
 version = "0.6.2"
```

Comment:
I would say that this should get an immediate release ... and I'm unsure whether patch or minor because technically the change to parametric types is breaking (especially for serialization), but I'm not sure how breaking it is in practice.

Reply:
Seems unlikely to be breaking in practice (why serialize one of the internal types here rather than just writing out the file) and nothing about the API has changed at all, so a non-breaking release should be okay.

Reply:
That was also my ultimate conclusion. :smile:

Reply:
```suggestion
version = "0.6.3"
```